website/src/components/HeaderBadges/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import BrowserOnly from '@docusaurus/BrowserOnly';
+
+export const HeaderBadges = () => {
```

Comment:
The easier we can make it for readers to provide feedback, the closer we get to having an actionable docs-specific feedback loop 🔥